invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -70,7 +70,7 @@ class AccessSettingsSchema(Schema):
     allow_user_requests = fields.Boolean()
     allow_guest_requests = fields.Boolean()
 
-    accept_conditions_text = SanitizedHTML()
+    accept_conditions_text = SanitizedHTML(allow_none=True)
 
 
 class ParentAccessSchema(Schema, FieldPermissionsMixin):
```

Comment:
shouldn't we add the settings in the "field_dump_permissions"?

Reply:
No, we need settings to be available for record serialization, for the request access form render. See here: https://github.com/inveniosoftware/invenio-app-rdm/blob/19b679f19d44623150333b1003b64e2710ca888a/invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html#L28C1-L28C1